src/components/Visualizations/Timeline.vue

```diff
@@ -592,13 +592,13 @@ export default {
         }
 
         context.shadowBlur = 0
-        context.strokeStyle = '#fff'
+        context.strokeStyle = '#fff' // this isn't working with a var, why?
```

Comment:
I think this might be similar to the bar chart so would using a computed style work here? Similar to what Nicholas mentioned above

Reply:
good call!